wagtail_wordpress_import/test/tests/test_transform_styles_filter.py

```diff
@@ -79,7 +79,7 @@ def test_transform_style_center(self):
         paragraph = soup.find("p")
 
         self.assertTrue(paragraph)
-        self.assertEqual(paragraph.attrs["class"], "align-center")
+        self.assertTrue("align-center" in paragraph.attrs["class"])
```

Comment:
**Comment**: I don't suppose it matters much in test case code, but there's `self.assertIn()`, too. e.g.

```suggestion
        self.assertIn("align-center", paragraph.attrs["class"])
```

Reply:
Nice